android/library/src/main/java/com/liferay/mobile/screens/util/AssetReader.java

```diff
@@ -55,7 +55,9 @@ private String getFileContent(InputStream inputStream) throws Exception {
 		String line = reader.readLine();
 
 		while (line != null) {
-			fileContent.append(line);
+			if (!line.startsWith("//")) {
```

Comment:
What if I use this method to read a plain text file?

Reply:
The name is maybe not the best, but we are using this class to read js and css files only :(

Reply:
then, renamed the method to something more specific

Reply:
👍